web/src/main/java/org/springframework/security/web/server/authentication/DelegatingServerAuthenticationConverter.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2002-2024 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.security.web.server.authentication;
+
+import java.util.List;
+import java.util.function.Function;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+import org.springframework.security.core.Authentication;
+import org.springframework.util.Assert;
+import org.springframework.web.server.ServerWebExchange;
+
+/**
+ * A {@link ServerAuthenticationConverter} that delegates to other
+ * {@link ServerAuthenticationConverter} instances.
+ *
+ * @author DingHao
+ * @since 6.3
+ */
+public class DelegatingServerAuthenticationConverter implements ServerAuthenticationConverter {
+
+	private final List<ServerAuthenticationConverter> delegates;
+
+	private boolean continueOnError = true;
```

Comment:
Please have this default to `false`. The reason is two-fold. First please see my comment [here](https://github.com/spring-projects/spring-security/pull/14654/files#r1506834943) and second I think it would be good to keep defaults aligned so that if we change them, they are changed together.

Reply:
Done